Source/driver/Castro.cpp

```diff
@@ -2345,8 +2345,8 @@ Castro::post_init (Real stop_time)
 
         if (sum_per > 0.0) {
 
-          const int num_per_old = floor((cumtime - dtlev) / sum_per);
-          const int num_per_new = floor((cumtime        ) / sum_per);
+          const int num_per_old = static_cast<int>(std::floor((cumtime - dtlev) / sum_per));
+          const int num_per_new = static_cast<int>(floor((cumtime        ) / sum_per));
```

Comment:
These should either both be std::floor or neither, right?

Reply:
I added the std::